app/models/price_group.rb

```diff
@@ -21,10 +21,12 @@ class PriceGroup < ApplicationRecord
   before_destroy { throw :abort if global? }
   before_destroy { price_policies.destroy_all } # Cannot be a dependent: :destroy because of ordering of callbacks
   before_create  ->(o) { o.display_order = 999 unless o.facility_id.nil? }
+  before_update  :update_hidden_price_group_policies, if: :will_save_change_to_is_hidden?
```

Comment:
What if I'm changing from hidden to visible? As the code is now, we'll make a call to set `can_purchase` to `false`, but that's unnecessary. I think we should avoid getting to that part of the code